lib/tasks/local_authorities.rake

```diff
@@ -0,0 +1,6 @@
+namespace :local_authorities do
+  desc "Import local authority names, codes and tiers from MapIt"
+  task import_codes: :environment do
```

Comment:
I've renamed these to `import_authorities`, `add_slugs` and `add_urls` which implies dependencies better